src/screens/Home/query.ts

```diff
@@ -10,6 +10,9 @@ export const QUERY_LATEST_POSTS = gql`
             }
             creation_date
             modification_date
+            replies {
+                id
```

Comment:
id is already returned, not sure what this does.

Reply:
just needed the amount of replies. so it's just counting the id objects in the array

Reply:
ok, this is hacky though. It should be achieved using a computed field. I'll add this to the table.
For the record, this is explained [here](https://docs.hasura.io/1.0/graphql/manual/schema/computed-fields.html#scalar-computed-fields) and I use the following SQL
```sql
CREATE FUNCTION replies_count(post_row posts)
RETURNS integer AS $$
  SELECT CAST( COUNT(*) as integer) FROM replies WHERE post_id = post_row.id
$$ LANGUAGE sql STABLE;
```
Ill comment here when it's usable

Reply:
Ok it's been merged https://github.com/Tbaut/hasura-auth/pull/10 
You will now have a `replies_count` field on `post`